src/stan/services/util/run_adaptive_sampler.hpp

```diff
@@ -66,7 +71,7 @@ void run_adaptive_sampler(Sampler& sampler, Model& model,
   auto start_warm = std::chrono::steady_clock::now();
   util::generate_transitions(sampler, num_warmup, 0, num_warmup + num_samples,
                              num_thin, refresh, save_warmup, true, writer, s,
-                             model, rng, interrupt, logger);
+                             model, rng, interrupt, logger, chain_id, n_chain);
```

Comment:
Sorry for the late catch...but why is it called `n_chain` here and not `num_chain`? `num_chain` looks more consistent with all the other variable conventions...would need to be addressed in the entire PR.